docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,284 @@ have not been included. These are:
+based on the QWERTY keyboard layout. These errors can include added characters, missed characters, 
+and replaced characters. In order to emulate that, there is a `GeCo like corrupter and related layout of the QWERTY keyboard csv found on github <https://github.com/ihmeuw/vivarium_research_prl/tree/main/noise>`_. 
+
+Currently, the user defined rate will be a rate 'eligible' for noise. The actual rate of noise 
+will be determined at the character level as XX errors per character. We think this approximates to 1 
+error per selected string, but obviously will not be exact. To implement this, select the strings 
+eligible for noise and then apply the OCR noise function to all strings. 
+
+Limitations: 
+
+- The result of having parameters at the character level is that we cannot tell exactly how many data points have noise in them. With the current settings, it is likely similar to the user selected value, but we will always have fewer errors than expected. This could be improved at a later time. 
+
+**Fake Names**
+
+For a variety of reasons, some repondents might choose to use a fake name rather 
+than their real name on official forms. To account for this, first select the sample 
+to have noise added. Then for everyone selected, replace their name with a random 
+selection from the `list of fake names here <https://github.com/ihmeuw/vivarium_research_prl/tree/main/noise>`_. 
+Please note that the list is separated into first and last names. 
+
+Limitations: 
+
+- Many of the fake first names include some information about the simulant (daughter, child f, minor) all specify something about the simulant. We will not try to match this information, which might lead to illogical information (an older man being labeled as 'daughter') but will not impact PRL. 
+- Someone who is likely to use a fake name might well do so across multiple observers. This would likely increase PRL challenges but will not be included here. 
+
+**Nicknames**
+
+Many people choose to use nicknames instead of their "real" names. A common example is an 
+Alexander who chooses to go by Alex. These individuals might write their nicknames on forms 
+which should be recorded. Here is a list of 1080 names and their `relevant nicknames <https://github.com/carltonnorthern/nicknames/blob/master/names.csv>`_. 
+
+Only those simulants with names in the csv above are eligible to recieve a nickname. First, 
+determine who is eligible for a nickname. Then select simulants for noise. Lastly, replace their 
+name with any of the nicknames included in the csv. If there are multiple options, 
+select at random. 
+
+.. note::
+
+  If the process of determining elgibility prior to selecting simulants for noise is challenging, we can work on finding a simpler approach. 
+
+
+**Miswriting** 
+
+For all numeric miswriting, first select the sample to have noise added. 
+Once the sample is selected, change 1 digit of the number at random to any 
```

Comment:
I haven't thought very carefully about how to do this well, but I was imagining we'd use a character-level probability of corruption like in the other noise functions.

And we might want different probabilities for different characters, depending on what the number represents. For example, for the WIC case study, I implemented a noise function for zipcodes that has a higher probability of corruption for the last 2 characters than for the first 3 characters, as it seems like the final two characters would be more likely to be wrong. Doing it that way should also be interesting for linkage since one blocking strategy is to block on the first 3 digits of zipcode.